prisma/schema.prisma

```diff
@@ -14,17 +14,51 @@ datasource db {
 }
 
 model Post {
```

Comment:
Let's change `Post` to `Event` at all locations in the schema. I believe this is supposed to be the model for an Event so let's make it consistent and appropriate.